File: 1-Leccion-Operaciones-Basicas/E.Dados-apilados.cs
using System.Collections.Generic;
using System.Linq;
using System;

class Program
{
  static void Main(string[] args)
  {
    int D = int.Parse(Console.ReadLine());
    
    int inferior = 14 * D + 1;
    int superior = 14 * D + 6;

    Console.WriteLine(string.Format("{0} {1}", inferior, superior));
  }
}

File: 1-Leccion-Operaciones-Basicas/E.Dados-apilados.cpp
#include <iostream>
using namespace std;

int main()
{
  int D;
	cin >> D;

  int inferior = D * 14 + 1;
	int superior = D * 14 + 6;	

  cout << inferior << " " << superior << endl;
  
  return 0;
}